Content/ClosePrintSpooler.md

```diff
@@ -0,0 +1,22 @@
+# Close Print Spooler
+
+_ClosePrintSpooler_ is a sandbox setting that provided nuanced control over how sandboxed applications interact with the print spooler service.
+
+```ini
+[DefaultBox]
+ClosePrintSpooler=n
+```
+
+This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
+
+Added as part of [0.5.4 / 5.46.0 version](https://github.com/sandboxie-plus/Sandboxie/blob/f3ab6a73a13a74f894b2d5ceb47ab69ee0581c97/CHANGELOG.md#054--5460---2021-01-06).
+
+## Interaction with [OpenPrintSpooler](OpenPrintSpooler.md)
+
+```ini
+[DefaultBox]
+ClosePrintSpooler=n
+OpenPrintSpooler=n
+```
```

Comment:
I think it is better to propose the new layout for all documentation settings at once in a separate pull request.

For now, please use the current layout:

https://github.com/sandboxie-plus/sandboxie-docs/blob/225d7162549d37a84b060001e9702e5ccff1a550/Content/AutoDelete.md?plain=1#L4-L10